apps/wallet-mobile/src/Dashboard/Dashboard.tsx

```diff
@@ -151,14 +152,32 @@ export const Dashboard = () => {
 const useNavigateTo = () => {
   const navigation = useNavigation()
 
+  if (CONFIG.DAPP_EXPLORER_ENABLED)
```

Comment:
That's a bit of a pain. To navigate to part of the staking center from outside this dashboard, this condition needs to be replicated?